setup.py

```diff
@@ -11,24 +9,20 @@
 
 
 requires = [
-    'requests',
+    'requests>=2.6.0',
 ]
 
-version = ''
-with open('telex/__init__.py', 'r') as fd:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        fd.read(), re.MULTILINE).group(1)
-
-with open('README.md', 'r', 'utf-8') as fd:
-    readme = fd.read()
-
+with open('README.md') as f:
+    readme = f.read()
+with open('LICENSE') as f:
+    license = f.read()
 
 setup(
     name='telex',
-    version=version,
+    version='0.0.1',
     description='Python telex client.',
     long_description=readme,
-    license='MIT',
+    license=license,
```

Comment:
As I mentioned above, this should stay as `MIT`. Otherwise, PyPI will be fail to render the information about the package.